plane/src/controller/command.rs

```diff
@@ -48,6 +48,7 @@ impl ControllerOpts {
             controller_url,
             default_cluster: self.default_cluster,
             cleanup_min_age_days: self.cleanup_min_age_days,
+            cleanup_batch_size: None,
```

Comment:
I feel like it doesn't make sense to store this value on the ControllerConfig and then thread it through so many functions if ultimately it can _only_ be `None`. Maybe we just have `run_cleanup_loop()` call `run_cleanup(&db, min_age_days, None)`? (And the db cli would continue to call `run_cleanup(&db, min_age_days, cleanup_batch_size)`.) Thoughts?

Reply:
The `None` here only applies to the simplified option set available when configuring Plane via the CLI. When run in production, we configure with a `.json` file which can express the full range of `ControllerConfig` functionality.

Reply:
Ohhh, gotcha gotcha